packages/vm/examples/eip7702-enable.ts

```diff
@@ -0,0 +1,115 @@
+  const implementationAddress = new Address(
+    hexToBytes('0x0000000000000000000000000000000000000123'),
+  )
+  console.log('Implementation address:', implementationAddress.toString())
+
+  // Create EIP-7702 transaction
+  // The authorization enables the EOA to use the code from the implementation address
+  const chainId = common.chainId()
+
+  // Create an authorization for EIP-7702
+  // This is normally signed by the EOA owner
+  const txData = {
+    nonce: 0n,
+    gasLimit: 100000n,
+    maxFeePerGas: 10000000000n, // 10 gwei
+    maxPriorityFeePerGas: 1000000000n, // 1 gwei
+    to: senderAddress, // target is the same as sender (self-call for initialization)
+    value: 0n,
+    data: hexToBytes('0x'), // Could be initialization data
+    accessList: [],
+    // Using just [] as a basic authorizationList that will be accepted
+    authorizationList: [],
+  }
+
+  // Create and sign the EIP-7702 transaction
+  const tx = createEOACode7702Tx(txData, { common })
+  const signedTx = tx.sign(privateKey)
+
+  console.log('Transaction type:', TransactionType[signedTx.type])
+  console.log('Supports EIP-7702:', signedTx.supports(Capability.EIP7702EOACode))
+  console.log('Transaction hash:', bytesToHex(signedTx.hash()))
+
+  // Run the transaction to set the code of the EOA
+  const result = await vm.runTx({ tx: signedTx })
+
+  console.log(
+    'Transaction processed:',
+    result.execResult.exceptionError !== null && result.execResult.exceptionError !== undefined
+      ? 'Failed'
+      : 'Success',
+  )
+
+  // Check that the EOA now has code
+  const account = await vm.stateManager.getAccount(senderAddress)
+  const code = await vm.stateManager.getContractCode(senderAddress)
+
+  console.log('Account nonce after transaction:', account?.nonce)
+  console.log('Account has code:', code.length > 0)
+  console.log('Code (hex):', bytesToHex(code))
+
+  // The code should start with the EIP-7702 delegation flag (0xef0100)
+  // followed by the implementation address
+  const hasDelegationFlag = code[0] === 0xef && code[1] === 0x01 && code[2] === 0x00
+  console.log('Has delegation flag:', hasDelegationFlag)
+
+  // Extract the delegated address from the code
+  if (hasDelegationFlag && code.length === 23) {
+    const delegatedAddressBytes = code.slice(3)
+    const delegatedAddress = createAddressFromString(bytesToHex(delegatedAddressBytes))
+    console.log('Delegated to address:', delegatedAddress.toString())
+
+    // Verify it matches our implementation address
+    console.log('Matches implementation address:', delegatedAddress.equals(implementationAddress))
+  }
+}
+
+// Helper function for bigint to unpadded bytes conversion
+function bigIntToUnpadded(value: bigint): Uint8Array {
+  // Convert bigint to bytes without padding
+  const hex = value.toString(16)
+  return hexToBytes(hex.length % 2 === 0 ? `0x${hex}` : `0x0${hex}`)
+}
+
+main().catch(console.error)
```

Comment:
Great work so far! Just chiming in regarding the potential failure of the examples CI job mentioned in #4016

```suggestion
main()
```

In order to test that the CI works for the examples job, the `catch` should be removed (or it should re-throw errors).